guides/common/modules/con_disaster-recovery-with-active-and-passive-project-server-and-external-storage.adoc

```diff
@@ -0,0 +1,6 @@
+[id="disaster-recovery-with-active-and-passive-{project-context}-server-and-external-storage"]
+= Disaster recovery with active and passive {ProjectServer} and external storage
+
+In this scenario, you maintain two {ProjectServer}s and you store critical data externally on network-attached storage (NAS).
+The primary server is active while the secondary server remains passive.
+If the primary server fails, you attach the NAS to your secondary server and turn it into your new primary server.
```

Comment:
I don't know the exact mechanism, but it might be that between this:

  `attach the NAS to your secondary server`

and this:

  `turn it into your new primary server`

some intermediate event(s) is/are happening, even if automatic. In that case it's worth mentioning, because otherwise this seems to me to have a gap in the sequence of events.

In addition to the potentially-existing event I mentioned, it might also make sense to explicitly say `make it active` right before the `and turn it into your new primary server` part.

Disregard these suggestions if you intended to only mention user actions here, though.